code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -100,13 +100,15 @@
 	initial_reagents = list(/decl/material/gas/ammonia = 60)
 
 /obj/item/chems/glass/bottle/eznutrient
+	name = "EZ NUtrient"
```

Comment:
This isn't a fix, these names will be cooked as soon as the holder updates from reagents.
The actual fix is making the biogenerator use the atom_info_repository instead of initial() in ui_interact() a la `atom_info_repository.get_name_for(path)`.

Reply:
This way it is displayed more pleasantly, and the cache will not store unnecessary information.
the second screenshot with my option
![image](https://user-images.githubusercontent.com/87154286/168331036-0d751959-6416-40e1-919c-a027c904e015.png)
![image](https://user-images.githubusercontent.com/87154286/168331202-c452e588-43b3-49db-b054-1c77f272b177.png)

Reply:
But the bottles themselves are going to be broken.

Reply:
You could use `initial(label_text)` instead of the atom repo, but ultimately you need to change it on the biogenerator not the bottle.

Reply:
Nvm, looks like `base_name` overrides name in Init so this is fine. Not a fan of the solution but biogenerator in general needs a rewrite so w/e.